oresat_configs/edl_cmd_defs.yaml

```diff
@@ -0,0 +1,235 @@
+    request:
+      - name: node_id
+        data_type: uint8
+        description: the id of The CANopen node to write to
+      - name: index
+        data_type: uint16
+        description: the OD index to write to
+      - name: subindex
+        data_type: uint8
+        description: the OD subindex to write to
+      - name: buffer
+        data_type: bytes
+        description: sdo data buffer to write
+        size_prefix: "uint32"
+    response:
+      - name: error_code
+        data_type: uint32
+        description: SDO error code (0 is no error)
+
+  - uid: 0x7
+    name: co_sync
+    description: send a CANopen SYNC message on the CAN bus
+    response:
+      - name: sent
+        data_type: bool
+        description: the CANopen SYNC message was sent successfully
+
+  - uid: 0x8
+    name: opd_sysenable
+    description: enable the OPD subsystem
+    request:
+      - name: enable
+        data_type: bool
+        description: true to enable or false to disable
+        enums:
+          disable: 0
+          enable: 1
+    response:
+      - name: status
+        data_type: bool
+        description: OPD subsystem enable status
+        enums:
+          disabled: 0
+          enabled: 1
+
+  - uid: 0x9
+    name: opd_scan
+    description: scan for all nodes on the OPD
+    response:
+      - name: nodes_found
+        data_type: uint8
+        description: the number of nodes found
+
+  - uid: 0xA
+    name: opd_probe
+    description: probe for a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to probe for
+    response:
+      - name: found
+        data_type: bool
+        description: true if the node was found or false if not
+        enums:
+          not_found: 0
+          found: 1
+
+  - uid: 0xB
+    name: opd_enable
+    description: enable / disable a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to enable / disable
+      - name: enable
+        data_type: bool
+        description: true to enable or false to disable
+        enums:
+          disable: 0
+          enable: 1
+    response:
+      - name: status
+        data_type: uint8
+        description: OPD node status
+
+  - uid: 0xC
+    name: opd_reset
+    description: reset a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: The id of the OPD node to reset
+    response:
+      - name: status
+        data_type: uint8
+        description: OPD node status
+
+  - uid: 0xD
+    name: opd_status
+    description: get the status of a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to get the status of
+    response:
+     - name: status
+       data_type: uint8
+       description: OPD node status
+
+  - uid: 0xE
+    name: rtc_set_time
```

Comment:
I know this is a bit bigger than this PR, but I would advocate for all commands to have one of either

* Always having an ack
* Always having some kind of response

In this case, maybe have the vehicle respond with the new rtc time it has?

Reply:
Make this an issue in oresat-c3-software

Reply:
All commands are the current one in c3-software as is, we can changes this after this PR and the c3 #37 PR are both merged.